x-pack/plugins/observability_ai_assistant/public/components/page_template.tsx

```diff
@@ -19,11 +19,7 @@ const pageSectionContentClassName = css`
 
 export function ObservabilityAIAssistantPageTemplate({ children }: { children: React.ReactNode }) {
   const {
-    services: {
```

Comment:
Had to do this as the `useLicense` hook needed the `license` plugin to be added to Obs Ai Assistant. 

When the Obs AI Assistant would be called from inside APM, it would crash, as APM adds plugin start dependencies not as `services: { plugins: { start: licensing }}}` but just `services: { licensing}`

Reply:
I don't think this  is the right way to go about it. We shouldn't depend on how plugins set their dependencies.

Reply:
I am slightly surprised that a Plugin which uses `useKibana` takes the plugin dependency structure from the app from which they are called...